athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/glue/GlueFieldLexer.java

```diff
@@ -111,6 +111,9 @@ else if (startToken.getValue().toLowerCase().equals(LIST)) {
         while (parser.hasNext() && parser.currentToken().getMarker() != GlueTypeParser.FIELD_END) {
             Field child = lex(parser.next(), parser, mapper);
             fieldBuilder.addField(child);
+            if (Types.getMinorTypeForArrowType(child.getType()) == Types.MinorType.LIST) {
+                parser.next();
```

Comment:
i think a comment will help to explain why we need to consume an extra token

Reply:
Added clarification comment in code.